homework/index.js

```diff
@@ -30,18 +15,89 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  function renderRepository(repository, parent) {
+    parent.innerHTML = '';
+    const table = createAndAppend('table', parent);
+    const tbody = createAndAppend('tbody', table);
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { text: 'Repository: ' });
+    const td = createAndAppend('td', tr);
+    createAndAppend('a', td, {
+      target: '_blank',
+      href: repository.html_url,
+      text: repository.name,
     });
+    const tr1 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr1, { text: 'Description: ' });
+    createAndAppend('td', tr1, { text: repository.description });
+
+    const tr2 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr2, { text: 'Forks: ' });
+    createAndAppend('td', tr2, { text: repository.forks });
+
+    const tr3 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr3, { text: 'Updated: ' });
+    const date = createAndAppend('td', tr3, { text: repository.updated_at });
+    date.innerHTML = new Date(repository.updated_at).toLocaleDateString();
   }
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  function renderContributors(contributors, parent) {
+    parent.innerHTML = '';
 
+    createAndAppend('p', parent, { text: 'Contributions' });
+
+    const ul = createAndAppend('ul', parent);
+    contributors.forEach(contributor => {
+      const li = createAndAppend('li', ul);
+      createAndAppend('img', li, { src: contributor.avatar_url });
+      const contributorInfo = createAndAppend('div', li, { id: 'contributor_info' });
```

Comment:
Element ID should be unique, you can use `class` if this ID is for styling purpose.